test/com/cdd/bao/template/SchemaTreeTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * BioAssay Ontology Annotator Tools
+ * 
+ * (c) 2017-2018 Collaborative Drug Discovery Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License 2.0
+ * as published by the Free Software Foundation:
+ * 
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.cdd.bao.template;
+
+import static org.junit.Assert.*;
+import static org.junit.Assume.*;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.*;
+import org.junit.*;
+
+import com.cdd.bao.template.*;
+import com.cdd.bao.template.Schema.*;
+import com.cdd.bao.template.Vocabulary.*;
+import com.cdd.bao.util.*;
+
+public class SchemaTreeTest extends OntologyReader
+{
+	private static class ProvTerm {
+		public String parentURI;
+		public String label;
+		public String descr;
+		public String uri;
+	}
+
+	private Vocabulary vocab;
+	private Schema schema;
+	private ProvTerm provTerm;
+
+	@Before
+	public void setUp() throws IOException
+	{
+		vocab = new Vocabulary();
+		vocab.loadExplicit(getPathsForTests(new String[]{"bao_complete_merged.owl"}));
```

Comment:
How long does it take to load the full vocabulary? If it takes a long time, would it be possible to have a smaller file that allows running the tests?